.eslintrc

```diff
@@ -1,5 +1,5 @@
 {
```

Comment:
Didn't mean to sneak this change in... My IDE was complaining about it not being valid JSON but it doesn't really matter.